Project.toml

```diff
@@ -1,22 +1,22 @@
 name = "CarlemanLinearization"
 uuid = "4803f6b2-022a-4c1b-a771-522a3413ec86"
-version = "0.2.1"
+version = "0.3.0"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MultivariatePolynomials = "102ac46a-7ee4-5c85-9060-abc95bfdeaa3"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-DynamicPolynomials = "0.3, 0.4"
+DynamicPolynomials = "0.4"
 MultivariatePolynomials = "0.3, 0.4"
-Requires = "0.5, 1"
+Requires = "1"
 julia = "1"
 
 [extras]
 DynamicPolynomials = "7c1d4256-1411-5781-91ec-d7bc3513ac07"
```

Comment:
Since `MultivariatePolynomials` is more specific, should we remove `DynamicPolynomials`? It is only used in a docstring.

Reply:
`DynamicPolynomials` is also used in the tests folder. In general, creating polynomial variables (eg. with the `@polyvar` macro) is provided by the implementations of the interface, not `MultivariatePolynomials`, so I don't think that we can remove it without sacrificing functionality tests.

Reply:
Okay, I did not check the tests. Then we keep it as is.